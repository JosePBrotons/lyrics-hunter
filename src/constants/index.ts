import theme, { COLORS } from './theme';

export { theme, COLORS };

export const SAVED_SEARCHES = '@SavedSearches:key'