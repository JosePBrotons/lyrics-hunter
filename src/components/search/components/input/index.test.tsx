import React from 'react';
import renderer from 'react-test-renderer';
import Input from '.';
import { COLORS } from '../../../../constants';

test('renders input component correctly', () => {
    const props = {
        placeholder: 'Artist',
        value: 'Coldplay',
        placeholderTextColor: COLORS.lightGray,
        onChangeText: jest.fn,
        iconName: 'user'
    };
    const inputCmp = renderer.create(<Input {...props} />).toJSON();
    expect(inputCmp).toMatchSnapshot();
});