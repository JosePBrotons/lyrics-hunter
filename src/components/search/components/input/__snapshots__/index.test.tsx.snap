// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders input component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F8F8F8",
      "borderRadius": 5,
      "color": "#454545",
      "flexDirection": "row",
      "fontSize": 14,
      "height": 40,
      "lineHeight": 22,
      "marginBottom": 20,
      "paddingHorizontal": 10,
      "width": "80%",
    }
  }
>
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": "#e9e9e9",
          "fontSize": 16,
        },
        undefined,
        Object {
          "fontFamily": "FontAwesome",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    ï€‡
  </Text>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Artist"
    placeholderTextColor="#e9e9e9"
    rejectResponderTermination={true}
    secureTextEntry={false}
    style={
      Object {
        "color": "#454545",
        "flex": 1,
        "paddingLeft": 8,
      }
    }
    underlineColorAndroid="transparent"
    value="Coldplay"
  />
</View>
`;
